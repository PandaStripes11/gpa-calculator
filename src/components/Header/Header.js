import React from 'react';

import logo from '@assets/logo.svg';

function Header() {
    return (
        <div className="header">
            <img className="logo" src={logo} alt="logo"/>
            <h1 className="title">GPA Calculator</h1>
        </div>
    )
}

export default Header