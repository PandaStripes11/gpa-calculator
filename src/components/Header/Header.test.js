import React from 'react';
import { shallow } from 'enzyme';
import Header from "./Header";

describe("<Header/>", () => {
    const header = shallow(<Header/>)
    test('has title of GPA Calculator', () => {
        expect(header.find('#title').text()).toEqual('GPA Calculator');
    });
    test('renders a .logo', () => {
        expect(header.find('.logo').exists()).toBeTruthy();
    });
});