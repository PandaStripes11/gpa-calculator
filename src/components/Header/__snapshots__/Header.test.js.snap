// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> matches snapshot 1`] = `
<header>
  <a
    className="title"
    href="."
  >
    <img
      alt="logo"
      className="logo"
      src="logo.svg"
    />
    <h1>
      GPA Calculator
    </h1>
  </a>
  <nav>
    <a
      href="."
    >
      Home
    </a>
    <a
      href="."
    >
      About
    </a>
    <a
      href="."
    >
      Contact
    </a>
    <a
      className="button-primary"
      href="."
    >
      Calculator
    </a>
  </nav>
</header>
`;
